dependencies.gradle

```diff
@@ -1,66 +1,31 @@
-// file:noinspection DependencyNotationArgument
+//file:noinspection DependencyNotationArgument
 // TODO remove when fixed in RFG ^
 dependencies {
     // Hard Dependencies
+
     // the CCL deobf jar uses very old MCP mappings, making it error at runtime in runClient/runServer
     // therefore we manually deobf the regular jar
-    implementation rfg.deobf("curse.maven:codechicken-lib-1-8-${ccl_pid}:${ccl_fid}")
+    implementation rfg.deobf("curse.maven:codechicken-lib-1-8-242818:2779848")
 
     // Soft Dependencies
-    implementation "mezz.jei:jei_1.12.2:${project.jei_version}"
-    implementation "CraftTweaker2:CraftTweaker2-MC1120-Main:1.12-${project.crt_version}"
-    implementation rfg.deobf("curse.maven:top-${top_pid}:${top_fid}")
-    implementation rfg.deobf("curse.maven:ctm-${ctm_pid}:${ctm_fid}")
+    // Can change any of these from compileOnlyApi -> implementation to test them in-game.
+
+    implementation "mezz.jei:jei_1.12.2:4.16.1.302"
+    implementation "CraftTweaker2:CraftTweaker2-MC1120-Main:1.12-4.1.20.684"
+    implementation rfg.deobf("curse.maven:top-245211:2667280") // TOP 1.4.28
+    implementation rfg.deobf("curse.maven:ctm-267602:2915363") // CTM 1.0.2.31
 
     // use a local jar for GroovyScript to avoid a bug from Mixin and ForgeGradle,
     // causing crashes at run-time when deobfuscated
     implementation files("libs/groovyscript-0.4.0.jar")
-    //implementation fg.deobf("curse.maven:groovyscript-${grs_pid}:${grs_fid}")
-
-    // Can change any of the below from compileOnlyApi -> implementation to test them in-game.
-    // Left as compileOnlyApi by default for a slimmer default instance.
-    compileOnlyApi rfg.deobf("curse.maven:ae2-extended-life-${ae2_pid}:${ae2_fid}")
-    compileOnlyApi rfg.deobf("curse.maven:opencomputers-${oc_pid}:${oc_fid}")
-    compileOnlyApi "curse.maven:journeymap-${jm_pid}:${jm_fid}"
-    compileOnlyApi "curse.maven:voxelmap-${voxel_pid}:${voxel_fid}"
-    compileOnlyApi "curse.maven:xaeros-${xaeros_pid}:${xaeros_fid}"
-
-    // Compile-Time Dependencies
-    annotationProcessor "com.github.bsideup.jabel:jabel-javac-plugin:${jabel_version}"
-    // workaround for https://github.com/bsideup/jabel/issues/174
-    annotationProcessor "net.java.dev.jna:jna-platform:${jna_version}"
-    compileOnly("com.github.bsideup.jabel:jabel-javac-plugin:${jabel_version}") {
-        transitive = false // We only care about the 1 annotation class
-    }
-    // Allow jdk.unsupported classes like sun.misc.Unsafe, workaround for JDK-8206937 and fixes Forge crashes in tests.
-    patchedMinecraft "me.eigenraven.java8unsupported:java-8-unsupported-shim:${unsupported_shim_version}"
-
-    // Tests
-    testImplementation "org.junit.jupiter:junit-jupiter:${junit_version}"
-    testImplementation "org.hamcrest:hamcrest:${hamcrest_version}"
+    //implementation fg.deobf("curse.maven:groovyscript-687577:4426112") // GRS 0.3.0
```

Comment:
Could you change this to `implementation rfg.deobf("curse.maven:groovyscript-687577:4487379") // GRS 0.4.6` so that we at least have an updated GS version commented out if we ever get it working